modules/utilities.js

```diff
@@ -46,8 +46,83 @@ const createHeroImage = opts => {
     };
 };
 
+// take a route config and format it for cloudfront
```

Comment:
Mildly wary of this becoming a dumping ground file. Maybe worth splitting this file up at some point.